From 74c090e74f26f9953953e21d252fdc8ff9a0be7d Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Thu, 22 Dec 2016 13:48:52 -0500
Subject: [PATCH 421/425] Ticket 49072 - fix log refactoring(part 2)

Description:  change slapi_log_err() to slapi_log_error()

https://fedorahosted.org/389/ticket/49072

Reviewed by: mreynolds(one line commit rule)

(cherry picked from commit 262f095f1b97a71b695d334bdd80ce35f11c4d07)
(cherry picked from commit f850128a0ba537dfab834205865b692f219da8ec)
---
 ldap/servers/plugins/memberof/memberof.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/servers/plugins/memberof/memberof.c b/ldap/servers/plugins/memberof/memberof.c
index 9f750f1..2cdaabb 100644
--- a/ldap/servers/plugins/memberof/memberof.c
+++ b/ldap/servers/plugins/memberof/memberof.c
@@ -2465,7 +2465,7 @@ int memberof_fix_memberof(MemberOfConfig *config, Slapi_Task *task, task_data *t
 
 		slapi_pblock_get(search_pb, SLAPI_PLUGIN_INTOP_RESULT, &result);
 		errmsg = ldap_err2string(result);
-		slapi_log_error(SLAPI_LOG_ERR, MEMBEROF_PLUGIN_SUBSYSTEM,
+		slapi_log_error(SLAPI_LOG_FATAL, MEMBEROF_PLUGIN_SUBSYSTEM,
 			"memberof_fix_memberof - Failed (%s)\n", errmsg );
 		slapi_task_log_notice(task, "Memberof task failed (%s)\n", errmsg );
 	}
-- 
2.9.3

